extensions/mapstore/pom.xml

```diff
@@ -144,6 +144,18 @@
             <version>12.4.0.jre8-preview</version>
             <scope>test</scope>
         </dependency>
+        <!--Oracle-->
+        <dependency>
+            <groupId>org.testcontainers</groupId>
+            <artifactId>oracle-xe</artifactId>
+            <version>${testcontainers.version}</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.oracle.database.jdbc</groupId>
+            <artifactId>ojdbc11</artifactId>
+            <version>23.2.0.0</version>
```

Comment:
This will be again in multiple modules, so we should have a property for it.

Reply:
absolutely, I was keeping that in mind